### PR TITLE
add initial table for feature status

### PR DESCRIPTION
**Pull request by inian**
## What is the new behavior?

CleanShot 2022-09-22

### CODE REVIEW

File: apps/reference/docs/architecture.mdx [hunk 1]
```diff
@@ -15,3 +15,25 @@ Supabase is not a 1-to-1 mapping of Firebase. While we are building many of the
...
+| Storage                    |                        | Beta  |
+| Storage                    | CDN                    | Beta  |
+| Edge Functions             |                        | Beta  |
+| Auth                       | OAuth Providers        | Beta  |
+| Auth                       | Passwordless           | Beta  |
+| Auth                       | Next.js Auth Helpers   | Alpha |
+| Auth                       | SvelteKit Auth Helpers | Alpha |
+| Public API                 |                        | Beta  |
+| CLI                        |                        | Beta  |
+| Client Library: JavaScript |                        | Beta  |
```
Comment (kiwicopple) [inline apps/reference/docs/architecture.mdx:38]: Should this lib be considered Beta? It's pretty stable, even before v2

File: apps/reference/docs/architecture.mdx [hunk 2]
```diff
@@ -15,3 +15,25 @@ Supabase is not a 1-to-1 mapping of Firebase. While we are building many of the
 Our technological choices are quite different from Firebase. Everything we use is open source. Wherever possible, we use and support existing tools rather than developing from scratch.
 
 Most notably, we use Postgres rather than a NoSQL store. This choice was deliberate. We believe that no other database offers the scalability and functionality required to compete with Firebase.
+
+## Feature Status
+
+| Product                    | Feature                | Stage |
```
Comment (kiwicopple) [inline apps/reference/docs/architecture.mdx:21]: should we add a column to link to the relevant docs (if available)?

**Review by kiwicopple — APPROVED** [review]

File: apps/reference/docs/architecture.mdx (hunk 1 above)
Comment (inian) [reply apps/reference/docs/architecture.mdx]: updated to GA

File: apps/reference/docs/architecture.mdx (hunk 2 above)
Comment (inian) [reply apps/reference/docs/architecture.mdx]: added

**Pull request merged**